controllers/profileController.js

```diff
@@ -55,7 +55,51 @@ exports.createFeatured = (req, res) => {
 }
 
 exports.createExperience = (req, res) => {
-  return
+  if (!req.body) {
+    return res.status(400).json({error: "Invalid request body"});
+  }
+  const newExperienceInfo = req.body;
+
+  if (!newExperienceInfo.user) {
+    return res.status(400).json({error: "User field is required"});
+  }
+
+  if (!newExperienceInfo.title) {
+    return res.status(400).json({error: "Title field is required"});
+  }
+
+  if (!newExperienceInfo.venue) {
+    return res.status(400).json({error: "Venue field is required"});
+  }
+
+  if (!newExperienceInfo.content) {
+    return res.status(400).json({error: "Content field is required"});
+  }
+
+  if (!newExperienceInfo.user) {
```

Comment:
Why are we checking the user here for the date? And shouldn't need a check because date is no longer required.